src/store/session.js

```diff
@@ -3,7 +3,7 @@ export default {
     state: {
         userInfo: null,
         appLanguage: localStorage.getItem('appLanguage') || 'no',
-        nightMode: JSON.parse(localStorage.getItem('nightMode')) || false,
+        nightMode: JSON.parse(localStorage.getItem('nightMode')) || (window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches) || false,
```

Comment:
`false || true || false` returns **true** while we want **false**, right?

Reply:
Good find, has been fixed